src/main/java/uk/gov/digital/ho/hocs/casework/client/infoclient/InfoClient.java

```diff
@@ -232,30 +185,20 @@ public void clearCachedTemplateForCaseType(String caseType) {
         return policies;
     }
 
-    @Cacheable(value = "InfoClientGetWorkingDaysElapsedForCaseType")
-    public Integer getWorkingDaysElapsedForCaseType(String caseType, LocalDate fromDate) {
-        String dateString = DateTimeFormatter.ofPattern("yyyy-MM-dd").format(fromDate);
-        Integer elapsedWorkingDays = restHelper.get(serviceBaseURL, String.format("/caseType/%s/workingDays/%s", caseType, dateString), new ParameterizedTypeReference<Integer>() {
-        });
-        log.info("Got working days elapsed for case type: {} fromDate: {}, event {}", caseType, dateString, value(EVENT, INFO_CLIENT_GET_WORKING_DAYS_FOR_CASE_TYPE_SUCCESS));
-        return elapsedWorkingDays;
-    }
-
-    public Integer getRemainingDaysToDeadline(String caseType, LocalDate deadlineDate) {
-        String dateString = DateTimeFormatter.ofPattern("yyyy-MM-dd").format(deadlineDate);
-        Integer remainingDays = restHelper.get(serviceBaseURL, String.format("/caseType/%s/deadline/%s/remainingDays", caseType, dateString), new ParameterizedTypeReference<Integer>() {
-        });
-        log.info("Got remaining days for caseType {} with deadline {} as {} days remaining, event {}", caseType, dateString, remainingDays, value(EVENT, INFO_CLIENT_REMAINING_DAYS_FOR_CASE_TYPE_AND_DEADLINE_SUCCESS));
-        return remainingDays;
-    }
-
     @Cacheable(value = "InfoGetProfileByCaseType", unless = "#result == null")
     public ProfileDto getProfileByCaseType(String caseType) {
         ProfileDto response = restHelper.get(serviceBaseURL, String.format("/profile/forcasetype/%s", caseType), ProfileDto.class);
         log.info("Got profile {} for case type {}, event {}", response.getProfileName(), caseType, value(EVENT, INFO_CLIENT_GET_PROFILE_BY_CASE_TYPE_SUCCESS));
         return response;
     }
 
+    @Cacheable(value = "InfoGetBankHolidaysByCaseType", unless = "#result == null")
```

Comment:
Presumably this won't be needed if we get the data locally

Reply:
This particular one will. It just returns the set of countries/regions whose bank holidays that apply to a case. For example, FOI would return ["ENGLAND_AND_WALES", "SCOTLAND", "NORTHERN_IRELAND"] as they are using all of the countries bank holidays. I left that particular config in info as it's data about a tenant.